boards/pinetime/Makefile.include

```diff
@@ -0,0 +1,7 @@
+# for this board, we are using Segger's RTT as default terminal interface
+USEMODULE += stdio_rtt
```

Comment:
Please have a look at #12724. That would help make things a little cleaner.

Reply:
Is this necessary if there's no periph_uart feature, which stdio_uart and ethos depend on?

Reply:
I think so.

This board is in the same case as the hamilton, so things related to stdio_rtt could be replaced with just:
```mk
# use JLink Segger RTT by default
RIOT_TERMINAL ?= jlink
include $(RIOTMAKE)/tools/serial.inc.mk
```

And add the following in `pinetime/Makefile.dep`:
```mk
# Use Segger's RTT by default for stdio on this board
DEFAULT_MODULE += stdio_rtt
```

Reply:
This looks weird: `include $(RIOTMAKE)/tools/serial.inc.mk`.
The board doesn't have uarts exposed...

Reply:
Ok, now that I tried I realized that in master, this doesn't work. ;(